h/streamer/messages.py

```diff
@@ -148,32 +159,29 @@ def _generate_annotation_event(
     if user_nipsad and socket.authenticated_userid != annotation.userid:
         return None
 
-    # The `prepare` function sets the active registry which is an implicit
-    # dependency of some of the authorization logic used to look up annotation
-    # and group permissions.
-    with pyramid.scripting.prepare(registry=socket.registry):
-        notification = {
-            "type": "annotation-notification",
-            "options": {"action": action},
-        }
-
-        base_url = socket.registry.settings.get("h.app_url", "http://localhost:5000")
-        links_service = LinksService(base_url, socket.registry)
-        resource = AnnotationNotificationContext(annotation, group_service, links_service)
-
-        # Check whether client is authorized to read this annotation.
-        read_principals = principals_allowed_by_permission(resource, "read")
-        if not set(read_principals).intersection(socket.effective_principals):
-            return None
-
-        serialized = presenters.AnnotationJSONPresenter(
-            resource, formatters=formatters
-        ).asdict()
-
-        notification["payload"] = [serialized]
-        if action == "delete":
-            notification["payload"] = [{"id": annotation.id}]
-        return notification
+    base_url = registry.settings.get("h.app_url", "http://localhost:5000")
+    links_service = LinksService(base_url, registry)
+    resource = AnnotationNotificationContext(annotation, group_service, links_service)
+
+    # Check whether client is authorized to read this annotation.
+    read_principals = principals_allowed_by_permission(resource, "read")
+    if not set(read_principals).intersection(socket.effective_principals):
+        return None
+
+    serialized = presenters.AnnotationJSONPresenter(
+        resource, formatters=formatters
+    ).asdict()
+
+    notification = {
+        "type": "annotation-notification",
+        "options": {"action": action},
+        "payload": [serialized],
+    }
```

Comment:
This is mostly the same code, just not indented, but I couldn't bring myself not to fix this part